synapse/telepath.py

```diff
@@ -195,6 +195,7 @@ class Proxy(s_coro.Fini):
 
     '''
     def __init__(self, link, name):
+        super().__init__()
```

Comment:
we've consistently avoided super() so far.  happy to discuss, but lets get it out of this delta so we can ninja-merge it :ninja: